test/fixtures/resources/PO026-assignVariable-propertySetRef/AM-AssignVariable-1.xml

```diff
@@ -0,0 +1,9 @@
+<AssignMessage name='AM-AssignVariable-1'>
+  <IgnoreUnresolvedVariables>false</IgnoreUnresolvedVariables>
+  <AssignVariable>
+    <!-- the PropertySeteRef element should contain a template
+         that isn't a single variable -->
+    <Name>invalid_propertysetref</Name>
+    <PropertySetRef>{curlies_are_wrong_here}</PropertySetRef>
```

Comment:
why is this wrong? I could have this or a combination of different variables like `{var1}-{var2}`

Reply:
I think PO026 is wrong in a different way.  The existing rule checks to see if there are curlies around the TEXT value, but ... after investigating today, it seems that is completely valid.  So the check, around line 105 of PO026-assignVariableUsage.js ,  should be removed. 

And to provide an explicit  answer to Sai's question, "why is this wrong":  It's not wrong.  It's ok to have curlies there.

Reply:
but per the PR, its wrong

Reply:
yes I understand. I think we should reject the PR and replace it with a different change, that removes the check for curlies. Curlies are fine within PropertySetRef!

Reply:
Yes